src/frontend/tsx/Chessboard.tsx

```diff
@@ -196,6 +196,12 @@ export function Chessboard () {
       fen: (displayIndex == null || displayGame.info.moves == null)
```

Comment:
Would you kindly please delete `boardTitle` (lines `60` - `64` inclusive) while you're making these changes.